src/components/Books/Books.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import AddBook from '../AddBook/AddBook';
+import Book from '../Book/Book';
+import './Books.css';
+
+function Books() {
+  return (
+    <div>
+      <div>
+        <div className="card"><Book title="Harry Potter" author="JK Rollings" category="Action" progress="64" chapter="Chapter 17" /></div>
+        <div className="card"><Book title="Avengers" author="Marvel" category="Science Fiction" progress="11" chapter="Chapter 3: A Lesson Learned" /></div>
+        <div className="card"><Book title="Cat in the hate" author="Some Donnei" category="Horror" progress="0" chapter="Introduction" /></div>
+      </div>
```

Comment:
In order to render dynamically, we will make use of the map method to iterate over an array of books.

You can take this model and implement it now. 
Or definitely, you will have to do it in the next milestones.

![image](https://user-images.githubusercontent.com/83596135/165979773-32aa3ce3-cf8f-4f57-910d-fe3fcad134fa.png)

Please, make sure to use semantic HTML containers. 
In your case, it should have been un `ul` tag with `li`s

Note: it's not a problem to deactivate linters for keys for now. We will learn about them later on;